cerberus-dashboard/src/components/CreateSDBoxForm/validator.js

```diff
@@ -18,7 +18,7 @@ import * as cms from '../../constants/cms';
 import { getLogger } from '../../utils/logger';
 var log = getLogger('create-new-sdb-validator');
 
-const doesContainNonAlphaNumericSpaceCharsRegex = /[^a-z\d\s]+/i;
+const doesContainNonAlphaNumericSpaceCharsRegex = /[^a-z\d\s\-]+/i;
```

Comment:
I don't think the '-' needs to be escaped.

Reply:
Removed it